conftest.py

```diff
@@ -2,12 +2,16 @@
 
 import os
 import sys
+from hashlib import md5
+
+import pytest
 
 pytest_plugins = [
     'sentry.utils.pytest'
 ]
 
 sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'src'))
+NUM_TEST_GROUPS = 2
```

Comment:
Should this be an env var defined in travis config?

Reply:
Yeah, I'm looking into ways to make this whole process more dynamic. Mostly just a POC to make sure it works, and is worthwhile (which it seems to be)